Source/Urho3D/Graphics/Texture.cpp

```diff
@@ -152,14 +152,6 @@ int Texture::GetLevelDepth(unsigned level) const
     return Max(depth_ >> level, 1);
 }
 
-unsigned Texture::GetDataSize(int width, int height) const
-{
-    if (IsCompressed())
-        return GetRowDataSize(width) * ((height + 3u) >> 2u);
-    else
-        return GetRowDataSize(width) * height;
-}
```

Comment:
Removal of this function seems wrong. Both directx backends include same function. OpenGL backend could override this function and implement pvrtc logic when necessary. In case texture is not pvrtc - call base function from `Texture` (this one that got removed).